src/traversals/graphvisit.jl

```diff
@@ -94,8 +94,8 @@ function close_vertex!(vis::LogGraphVisitor, v::Int)
     return true
 end
 
-function examine_neighbor!(vis::LogGraphVisitor, u::Int, v::Int, vcolor::Int, ecolor::Int)
-    println(vis.io, "examine neighbor: $u -> $v (vertexcolor = $vcolor, edgecolor= $ecolor)")
+function examine_neighbor!(vis::LogGraphVisitor, u::Int, v::Int, ucolor::Int, vcolor::Int, ecolor::Int)
+    println(vis.io, "examine neighbor: $u -> $v (headvertexcolor = $ucolor, tailvertexcolor = $vcolor, edgecolor= $ecolor)")
```

Comment:
Is this really `srcvertexcolor` and `dstvertexcolor`? Not to bikeshed the name, but we use `u` and `v` to denote `src` and `dst` elsewhere.